core-services/release-controller/_releases/release-ocp-4.9-ppc64le.json

```diff
@@ -13,5 +13,9 @@
     "tag":{"tagRef":{"name":"4.9"}}
   },
   "verify":{
+    "ppc64le":{
+  "optional":true,
+  "prowJob":{"name":"periodic-ci-openshift-multiarch-master-nightly-4.9-installer-remote-libvirt-ppc64le"}
```

Comment:
This is formatted in a super weird way - why?

Reply:
Missed indentation. Fixed it!